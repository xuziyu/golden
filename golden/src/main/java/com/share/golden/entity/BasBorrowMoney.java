package com.share.golden.entity;

public class BasBorrowMoney {
    private String borrowId;

    private String borrowSerial;

    private String userId;

    private String borrowCompany;

    private String borrowMoney;

    private String borrowRate;

    private String timeLimit;

    private String borrowLimit;

    private String borrowState;

    private String borrowRecommend;

    private String borrowType;

    private String borrowRemark1;

    private String borrowRemark2;

    public String getBorrowId() {
        return borrowId;
    }

    public void setBorrowId(String borrowId) {
        this.borrowId = borrowId == null ? null : borrowId.trim();
    }

    public String getBorrowSerial() {
        return borrowSerial;
    }

    public void setBorrowSerial(String borrowSerial) {
        this.borrowSerial = borrowSerial == null ? null : borrowSerial.trim();
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId == null ? null : userId.trim();
    }

    public String getBorrowCompany() {
        return borrowCompany;
    }

    public void setBorrowCompany(String borrowCompany) {
        this.borrowCompany = borrowCompany == null ? null : borrowCompany.trim();
    }

    public String getBorrowMoney() {
        return borrowMoney;
    }

    public void setBorrowMoney(String borrowMoney) {
        this.borrowMoney = borrowMoney == null ? null : borrowMoney.trim();
    }

    public String getBorrowRate() {
        return borrowRate;
    }

    public void setBorrowRate(String borrowRate) {
        this.borrowRate = borrowRate == null ? null : borrowRate.trim();
    }

    public String getTimeLimit() {
        return timeLimit;
    }

    public void setTimeLimit(String timeLimit) {
        this.timeLimit = timeLimit == null ? null : timeLimit.trim();
    }

    public String getBorrowLimit() {
        return borrowLimit;
    }

    public void setBorrowLimit(String borrowLimit) {
        this.borrowLimit = borrowLimit == null ? null : borrowLimit.trim();
    }

    public String getBorrowState() {
        return borrowState;
    }

    public void setBorrowState(String borrowState) {
        this.borrowState = borrowState == null ? null : borrowState.trim();
    }

    public String getBorrowRecommend() {
        return borrowRecommend;
    }

    public void setBorrowRecommend(String borrowRecommend) {
        this.borrowRecommend = borrowRecommend == null ? null : borrowRecommend.trim();
    }

    public String getBorrowType() {
        return borrowType;
    }

    public void setBorrowType(String borrowType) {
        this.borrowType = borrowType == null ? null : borrowType.trim();
    }

    public String getBorrowRemark1() {
        return borrowRemark1;
    }

    public void setBorrowRemark1(String borrowRemark1) {
        this.borrowRemark1 = borrowRemark1 == null ? null : borrowRemark1.trim();
    }

    public String getBorrowRemark2() {
        return borrowRemark2;
    }

    public void setBorrowRemark2(String borrowRemark2) {
        this.borrowRemark2 = borrowRemark2 == null ? null : borrowRemark2.trim();
    }
}