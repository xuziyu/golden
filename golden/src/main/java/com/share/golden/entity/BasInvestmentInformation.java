package com.share.golden.entity;

import java.math.BigDecimal;
import java.util.Date;

public class BasInvestmentInformation {
    private String investorId;

    private String userId;

    private String tenderId;

    private BigDecimal investorMoney;

    private String investorStatus;

    private String investorStyle;

    private String brrowStatus;

    private String investorRest;

    private String profitModel;

    private BigDecimal profitMoney;

    private Date investorDate;

    private String replayDate;

    private String investorType;

    public String getInvestorId() {
        return investorId;
    }

    public void setInvestorId(String investorId) {
        this.investorId = investorId == null ? null : investorId.trim();
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId == null ? null : userId.trim();
    }

    public String getTenderId() {
        return tenderId;
    }

    public void setTenderId(String tenderId) {
        this.tenderId = tenderId == null ? null : tenderId.trim();
    }

    public BigDecimal getInvestorMoney() {
        return investorMoney;
    }

    public void setInvestorMoney(BigDecimal investorMoney) {
        this.investorMoney = investorMoney;
    }

    public String getInvestorStatus() {
        return investorStatus;
    }

    public void setInvestorStatus(String investorStatus) {
        this.investorStatus = investorStatus == null ? null : investorStatus.trim();
    }

    public String getInvestorStyle() {
        return investorStyle;
    }

    public void setInvestorStyle(String investorStyle) {
        this.investorStyle = investorStyle == null ? null : investorStyle.trim();
    }

    public String getBrrowStatus() {
        return brrowStatus;
    }

    public void setBrrowStatus(String brrowStatus) {
        this.brrowStatus = brrowStatus == null ? null : brrowStatus.trim();
    }

    public String getInvestorRest() {
        return investorRest;
    }

    public void setInvestorRest(String investorRest) {
        this.investorRest = investorRest == null ? null : investorRest.trim();
    }

    public String getProfitModel() {
        return profitModel;
    }

    public void setProfitModel(String profitModel) {
        this.profitModel = profitModel == null ? null : profitModel.trim();
    }

    public BigDecimal getProfitMoney() {
        return profitMoney;
    }

    public void setProfitMoney(BigDecimal profitMoney) {
        this.profitMoney = profitMoney;
    }

    public Date getInvestorDate() {
        return investorDate;
    }

    public void setInvestorDate(Date investorDate) {
        this.investorDate = investorDate;
    }

    public String getReplayDate() {
        return replayDate;
    }

    public void setReplayDate(String replayDate) {
        this.replayDate = replayDate == null ? null : replayDate.trim();
    }

    public String getInvestorType() {
        return investorType;
    }

    public void setInvestorType(String investorType) {
        this.investorType = investorType == null ? null : investorType.trim();
    }
}