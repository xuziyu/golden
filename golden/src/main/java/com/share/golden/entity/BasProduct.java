package com.share.golden.entity;

import java.math.BigDecimal;
import java.util.Date;

public class BasProduct {
    private String productId;

    private BigDecimal productMoney;

    private String productIncome;

    private Date productTime;

    private String productType;

    private String productWay;

    private Date pcount;

    private String productProgress;

    private String productSaveway;

    private String productRateben;

    private Date productPublishTime;

    private String productName;

    private BigDecimal productTotalMoney;

    private String investmentScope;

    private String useOfProceeds;

    private String productState;

    private String productPicture;

    private String produceId;

    private String productDesc;

    private String productSafe;

    public String getProductId() {
        return productId;
    }

    public void setProductId(String productId) {
        this.productId = productId == null ? null : productId.trim();
    }

    public BigDecimal getProductMoney() {
        return productMoney;
    }

    public void setProductMoney(BigDecimal productMoney) {
        this.productMoney = productMoney;
    }

    public String getProductIncome() {
        return productIncome;
    }

    public void setProductIncome(String productIncome) {
        this.productIncome = productIncome == null ? null : productIncome.trim();
    }

    public Date getProductTime() {
        return productTime;
    }

    public void setProductTime(Date productTime) {
        this.productTime = productTime;
    }

    public String getProductType() {
        return productType;
    }

    public void setProductType(String productType) {
        this.productType = productType == null ? null : productType.trim();
    }

    public String getProductWay() {
        return productWay;
    }

    public void setProductWay(String productWay) {
        this.productWay = productWay == null ? null : productWay.trim();
    }

    public Date getPcount() {
        return pcount;
    }

    public void setPcount(Date pcount) {
        this.pcount = pcount;
    }

    public String getProductProgress() {
        return productProgress;
    }

    public void setProductProgress(String productProgress) {
        this.productProgress = productProgress == null ? null : productProgress.trim();
    }

    public String getProductSaveway() {
        return productSaveway;
    }

    public void setProductSaveway(String productSaveway) {
        this.productSaveway = productSaveway == null ? null : productSaveway.trim();
    }

    public String getProductRateben() {
        return productRateben;
    }

    public void setProductRateben(String productRateben) {
        this.productRateben = productRateben == null ? null : productRateben.trim();
    }

    public Date getProductPublishTime() {
        return productPublishTime;
    }

    public void setProductPublishTime(Date productPublishTime) {
        this.productPublishTime = productPublishTime;
    }

    public String getProductName() {
        return productName;
    }

    public void setProductName(String productName) {
        this.productName = productName == null ? null : productName.trim();
    }

    public BigDecimal getProductTotalMoney() {
        return productTotalMoney;
    }

    public void setProductTotalMoney(BigDecimal productTotalMoney) {
        this.productTotalMoney = productTotalMoney;
    }

    public String getInvestmentScope() {
        return investmentScope;
    }

    public void setInvestmentScope(String investmentScope) {
        this.investmentScope = investmentScope == null ? null : investmentScope.trim();
    }

    public String getUseOfProceeds() {
        return useOfProceeds;
    }

    public void setUseOfProceeds(String useOfProceeds) {
        this.useOfProceeds = useOfProceeds == null ? null : useOfProceeds.trim();
    }

    public String getProductState() {
        return productState;
    }

    public void setProductState(String productState) {
        this.productState = productState == null ? null : productState.trim();
    }

    public String getProductPicture() {
        return productPicture;
    }

    public void setProductPicture(String productPicture) {
        this.productPicture = productPicture == null ? null : productPicture.trim();
    }

    public String getProduceId() {
        return produceId;
    }

    public void setProduceId(String produceId) {
        this.produceId = produceId == null ? null : produceId.trim();
    }

    public String getProductDesc() {
        return productDesc;
    }

    public void setProductDesc(String productDesc) {
        this.productDesc = productDesc == null ? null : productDesc.trim();
    }

    public String getProductSafe() {
        return productSafe;
    }

    public void setProductSafe(String productSafe) {
        this.productSafe = productSafe == null ? null : productSafe.trim();
    }
}