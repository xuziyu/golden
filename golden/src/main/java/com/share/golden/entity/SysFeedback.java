package com.share.golden.entity;

import java.util.Date;

public class SysFeedback {
    private String feedbackId;

    private String feedbackName;

    private String feedbackConent;

    private Date feedbackTime;

    public String getFeedbackId() {
        return feedbackId;
    }

    public void setFeedbackId(String feedbackId) {
        this.feedbackId = feedbackId == null ? null : feedbackId.trim();
    }

    public String getFeedbackName() {
        return feedbackName;
    }

    public void setFeedbackName(String feedbackName) {
        this.feedbackName = feedbackName == null ? null : feedbackName.trim();
    }

    public String getFeedbackConent() {
        return feedbackConent;
    }

    public void setFeedbackConent(String feedbackConent) {
        this.feedbackConent = feedbackConent == null ? null : feedbackConent.trim();
    }

    public Date getFeedbackTime() {
        return feedbackTime;
    }

    public void setFeedbackTime(Date feedbackTime) {
        this.feedbackTime = feedbackTime;
    }
}