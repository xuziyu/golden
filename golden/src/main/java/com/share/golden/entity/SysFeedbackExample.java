package com.share.golden.entity;

import java.util.ArrayList;
import java.util.Date;
import java.util.List;

public class SysFeedbackExample {
    protected String orderByClause;

    protected boolean distinct;

    protected List<Criteria> oredCriteria;

    public SysFeedbackExample() {
        oredCriteria = new ArrayList<Criteria>();
    }

    public void setOrderByClause(String orderByClause) {
        this.orderByClause = orderByClause;
    }

    public String getOrderByClause() {
        return orderByClause;
    }

    public void setDistinct(boolean distinct) {
        this.distinct = distinct;
    }

    public boolean isDistinct() {
        return distinct;
    }

    public List<Criteria> getOredCriteria() {
        return oredCriteria;
    }

    public void or(Criteria criteria) {
        oredCriteria.add(criteria);
    }

    public Criteria or() {
        Criteria criteria = createCriteriaInternal();
        oredCriteria.add(criteria);
        return criteria;
    }

    public Criteria createCriteria() {
        Criteria criteria = createCriteriaInternal();
        if (oredCriteria.size() == 0) {
            oredCriteria.add(criteria);
        }
        return criteria;
    }

    protected Criteria createCriteriaInternal() {
        Criteria criteria = new Criteria();
        return criteria;
    }

    public void clear() {
        oredCriteria.clear();
        orderByClause = null;
        distinct = false;
    }

    protected abstract static class GeneratedCriteria {
        protected List<Criterion> criteria;

        protected GeneratedCriteria() {
            super();
            criteria = new ArrayList<Criterion>();
        }

        public boolean isValid() {
            return criteria.size() > 0;
        }

        public List<Criterion> getAllCriteria() {
            return criteria;
        }

        public List<Criterion> getCriteria() {
            return criteria;
        }

        protected void addCriterion(String condition) {
            if (condition == null) {
                throw new RuntimeException("Value for condition cannot be null");
            }
            criteria.add(new Criterion(condition));
        }

        protected void addCriterion(String condition, Object value, String property) {
            if (value == null) {
                throw new RuntimeException("Value for " + property + " cannot be null");
            }
            criteria.add(new Criterion(condition, value));
        }

        protected void addCriterion(String condition, Object value1, Object value2, String property) {
            if (value1 == null || value2 == null) {
                throw new RuntimeException("Between values for " + property + " cannot be null");
            }
            criteria.add(new Criterion(condition, value1, value2));
        }

        public Criteria andFeedbackIdIsNull() {
            addCriterion("\"FEEDBACK_ID\" is null");
            return (Criteria) this;
        }

        public Criteria andFeedbackIdIsNotNull() {
            addCriterion("\"FEEDBACK_ID\" is not null");
            return (Criteria) this;
        }

        public Criteria andFeedbackIdEqualTo(String value) {
            addCriterion("\"FEEDBACK_ID\" =", value, "feedbackId");
            return (Criteria) this;
        }

        public Criteria andFeedbackIdNotEqualTo(String value) {
            addCriterion("\"FEEDBACK_ID\" <>", value, "feedbackId");
            return (Criteria) this;
        }

        public Criteria andFeedbackIdGreaterThan(String value) {
            addCriterion("\"FEEDBACK_ID\" >", value, "feedbackId");
            return (Criteria) this;
        }

        public Criteria andFeedbackIdGreaterThanOrEqualTo(String value) {
            addCriterion("\"FEEDBACK_ID\" >=", value, "feedbackId");
            return (Criteria) this;
        }

        public Criteria andFeedbackIdLessThan(String value) {
            addCriterion("\"FEEDBACK_ID\" <", value, "feedbackId");
            return (Criteria) this;
        }

        public Criteria andFeedbackIdLessThanOrEqualTo(String value) {
            addCriterion("\"FEEDBACK_ID\" <=", value, "feedbackId");
            return (Criteria) this;
        }

        public Criteria andFeedbackIdLike(String value) {
            addCriterion("\"FEEDBACK_ID\" like", value, "feedbackId");
            return (Criteria) this;
        }

        public Criteria andFeedbackIdNotLike(String value) {
            addCriterion("\"FEEDBACK_ID\" not like", value, "feedbackId");
            return (Criteria) this;
        }

        public Criteria andFeedbackIdIn(List<String> values) {
            addCriterion("\"FEEDBACK_ID\" in", values, "feedbackId");
            return (Criteria) this;
        }

        public Criteria andFeedbackIdNotIn(List<String> values) {
            addCriterion("\"FEEDBACK_ID\" not in", values, "feedbackId");
            return (Criteria) this;
        }

        public Criteria andFeedbackIdBetween(String value1, String value2) {
            addCriterion("\"FEEDBACK_ID\" between", value1, value2, "feedbackId");
            return (Criteria) this;
        }

        public Criteria andFeedbackIdNotBetween(String value1, String value2) {
            addCriterion("\"FEEDBACK_ID\" not between", value1, value2, "feedbackId");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameIsNull() {
            addCriterion("\"FEEDBACK_NAME\" is null");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameIsNotNull() {
            addCriterion("\"FEEDBACK_NAME\" is not null");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameEqualTo(String value) {
            addCriterion("\"FEEDBACK_NAME\" =", value, "feedbackName");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameNotEqualTo(String value) {
            addCriterion("\"FEEDBACK_NAME\" <>", value, "feedbackName");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameGreaterThan(String value) {
            addCriterion("\"FEEDBACK_NAME\" >", value, "feedbackName");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameGreaterThanOrEqualTo(String value) {
            addCriterion("\"FEEDBACK_NAME\" >=", value, "feedbackName");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameLessThan(String value) {
            addCriterion("\"FEEDBACK_NAME\" <", value, "feedbackName");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameLessThanOrEqualTo(String value) {
            addCriterion("\"FEEDBACK_NAME\" <=", value, "feedbackName");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameLike(String value) {
            addCriterion("\"FEEDBACK_NAME\" like", value, "feedbackName");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameNotLike(String value) {
            addCriterion("\"FEEDBACK_NAME\" not like", value, "feedbackName");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameIn(List<String> values) {
            addCriterion("\"FEEDBACK_NAME\" in", values, "feedbackName");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameNotIn(List<String> values) {
            addCriterion("\"FEEDBACK_NAME\" not in", values, "feedbackName");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameBetween(String value1, String value2) {
            addCriterion("\"FEEDBACK_NAME\" between", value1, value2, "feedbackName");
            return (Criteria) this;
        }

        public Criteria andFeedbackNameNotBetween(String value1, String value2) {
            addCriterion("\"FEEDBACK_NAME\" not between", value1, value2, "feedbackName");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentIsNull() {
            addCriterion("\"FEEDBACK_CONENT\" is null");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentIsNotNull() {
            addCriterion("\"FEEDBACK_CONENT\" is not null");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentEqualTo(String value) {
            addCriterion("\"FEEDBACK_CONENT\" =", value, "feedbackConent");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentNotEqualTo(String value) {
            addCriterion("\"FEEDBACK_CONENT\" <>", value, "feedbackConent");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentGreaterThan(String value) {
            addCriterion("\"FEEDBACK_CONENT\" >", value, "feedbackConent");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentGreaterThanOrEqualTo(String value) {
            addCriterion("\"FEEDBACK_CONENT\" >=", value, "feedbackConent");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentLessThan(String value) {
            addCriterion("\"FEEDBACK_CONENT\" <", value, "feedbackConent");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentLessThanOrEqualTo(String value) {
            addCriterion("\"FEEDBACK_CONENT\" <=", value, "feedbackConent");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentLike(String value) {
            addCriterion("\"FEEDBACK_CONENT\" like", value, "feedbackConent");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentNotLike(String value) {
            addCriterion("\"FEEDBACK_CONENT\" not like", value, "feedbackConent");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentIn(List<String> values) {
            addCriterion("\"FEEDBACK_CONENT\" in", values, "feedbackConent");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentNotIn(List<String> values) {
            addCriterion("\"FEEDBACK_CONENT\" not in", values, "feedbackConent");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentBetween(String value1, String value2) {
            addCriterion("\"FEEDBACK_CONENT\" between", value1, value2, "feedbackConent");
            return (Criteria) this;
        }

        public Criteria andFeedbackConentNotBetween(String value1, String value2) {
            addCriterion("\"FEEDBACK_CONENT\" not between", value1, value2, "feedbackConent");
            return (Criteria) this;
        }

        public Criteria andFeedbackTimeIsNull() {
            addCriterion("\"FEEDBACK_TIME\" is null");
            return (Criteria) this;
        }

        public Criteria andFeedbackTimeIsNotNull() {
            addCriterion("\"FEEDBACK_TIME\" is not null");
            return (Criteria) this;
        }

        public Criteria andFeedbackTimeEqualTo(Date value) {
            addCriterion("\"FEEDBACK_TIME\" =", value, "feedbackTime");
            return (Criteria) this;
        }

        public Criteria andFeedbackTimeNotEqualTo(Date value) {
            addCriterion("\"FEEDBACK_TIME\" <>", value, "feedbackTime");
            return (Criteria) this;
        }

        public Criteria andFeedbackTimeGreaterThan(Date value) {
            addCriterion("\"FEEDBACK_TIME\" >", value, "feedbackTime");
            return (Criteria) this;
        }

        public Criteria andFeedbackTimeGreaterThanOrEqualTo(Date value) {
            addCriterion("\"FEEDBACK_TIME\" >=", value, "feedbackTime");
            return (Criteria) this;
        }

        public Criteria andFeedbackTimeLessThan(Date value) {
            addCriterion("\"FEEDBACK_TIME\" <", value, "feedbackTime");
            return (Criteria) this;
        }

        public Criteria andFeedbackTimeLessThanOrEqualTo(Date value) {
            addCriterion("\"FEEDBACK_TIME\" <=", value, "feedbackTime");
            return (Criteria) this;
        }

        public Criteria andFeedbackTimeIn(List<Date> values) {
            addCriterion("\"FEEDBACK_TIME\" in", values, "feedbackTime");
            return (Criteria) this;
        }

        public Criteria andFeedbackTimeNotIn(List<Date> values) {
            addCriterion("\"FEEDBACK_TIME\" not in", values, "feedbackTime");
            return (Criteria) this;
        }

        public Criteria andFeedbackTimeBetween(Date value1, Date value2) {
            addCriterion("\"FEEDBACK_TIME\" between", value1, value2, "feedbackTime");
            return (Criteria) this;
        }

        public Criteria andFeedbackTimeNotBetween(Date value1, Date value2) {
            addCriterion("\"FEEDBACK_TIME\" not between", value1, value2, "feedbackTime");
            return (Criteria) this;
        }
    }

    public static class Criteria extends GeneratedCriteria {

        protected Criteria() {
            super();
        }
    }

    public static class Criterion {
        private String condition;

        private Object value;

        private Object secondValue;

        private boolean noValue;

        private boolean singleValue;

        private boolean betweenValue;

        private boolean listValue;

        private String typeHandler;

        public String getCondition() {
            return condition;
        }

        public Object getValue() {
            return value;
        }

        public Object getSecondValue() {
            return secondValue;
        }

        public boolean isNoValue() {
            return noValue;
        }

        public boolean isSingleValue() {
            return singleValue;
        }

        public boolean isBetweenValue() {
            return betweenValue;
        }

        public boolean isListValue() {
            return listValue;
        }

        public String getTypeHandler() {
            return typeHandler;
        }

        protected Criterion(String condition) {
            super();
            this.condition = condition;
            this.typeHandler = null;
            this.noValue = true;
        }

        protected Criterion(String condition, Object value, String typeHandler) {
            super();
            this.condition = condition;
            this.value = value;
            this.typeHandler = typeHandler;
            if (value instanceof List<?>) {
                this.listValue = true;
            } else {
                this.singleValue = true;
            }
        }

        protected Criterion(String condition, Object value) {
            this(condition, value, null);
        }

        protected Criterion(String condition, Object value, Object secondValue, String typeHandler) {
            super();
            this.condition = condition;
            this.value = value;
            this.secondValue = secondValue;
            this.typeHandler = typeHandler;
            this.betweenValue = true;
        }

        protected Criterion(String condition, Object value, Object secondValue) {
            this(condition, value, secondValue, null);
        }
    }
}