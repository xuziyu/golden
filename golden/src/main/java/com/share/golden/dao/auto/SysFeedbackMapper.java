package com.share.golden.dao.auto;

import com.share.golden.entity.SysFeedback;
import com.share.golden.entity.SysFeedbackExample;
import java.util.List;
import org.apache.ibatis.annotations.Param;

public interface SysFeedbackMapper {
    long countByExample(SysFeedbackExample example);

    int deleteByExample(SysFeedbackExample example);

    int deleteByPrimaryKey(String feedbackId);

    int insert(SysFeedback record);

    int insertSelective(SysFeedback record);

    List<SysFeedback> selectByExample(SysFeedbackExample example);

    SysFeedback selectByPrimaryKey(String feedbackId);

    int updateByExampleSelective(@Param("record") SysFeedback record, @Param("example") SysFeedbackExample example);

    int updateByExample(@Param("record") SysFeedback record, @Param("example") SysFeedbackExample example);

    int updateByPrimaryKeySelective(SysFeedback record);

    int updateByPrimaryKey(SysFeedback record);
}