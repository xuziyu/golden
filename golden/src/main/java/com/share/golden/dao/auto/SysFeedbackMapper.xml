<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
<mapper namespace="com.share.golden.dao.auto.SysFeedbackMapper">
  <resultMap id="BaseResultMap" type="com.share.golden.entity.SysFeedback">
    <id column="FEEDBACK_ID" jdbcType="VARCHAR" property="feedbackId" />
    <result column="FEEDBACK_NAME" jdbcType="VARCHAR" property="feedbackName" />
    <result column="FEEDBACK_CONENT" jdbcType="VARCHAR" property="feedbackConent" />
    <result column="FEEDBACK_TIME" jdbcType="TIMESTAMP" property="feedbackTime" />
  </resultMap>
  <sql id="Example_Where_Clause">
    <where>
      <foreach collection="oredCriteria" item="criteria" separator="or">
        <if test="criteria.valid">
          <trim prefix="(" prefixOverrides="and" suffix=")">
            <foreach collection="criteria.criteria" item="criterion">
              <choose>
                <when test="criterion.noValue">
                  and ${criterion.condition}
                </when>
                <when test="criterion.singleValue">
                  and ${criterion.condition} #{criterion.value}
                </when>
                <when test="criterion.betweenValue">
                  and ${criterion.condition} #{criterion.value} and #{criterion.secondValue}
                </when>
                <when test="criterion.listValue">
                  and ${criterion.condition}
                  <foreach close=")" collection="criterion.value" item="listItem" open="(" separator=",">
                    #{listItem}
                  </foreach>
                </when>
              </choose>
            </foreach>
          </trim>
        </if>
      </foreach>
    </where>
  </sql>
  <sql id="Update_By_Example_Where_Clause">
    <where>
      <foreach collection="example.oredCriteria" item="criteria" separator="or">
        <if test="criteria.valid">
          <trim prefix="(" prefixOverrides="and" suffix=")">
            <foreach collection="criteria.criteria" item="criterion">
              <choose>
                <when test="criterion.noValue">
                  and ${criterion.condition}
                </when>
                <when test="criterion.singleValue">
                  and ${criterion.condition} #{criterion.value}
                </when>
                <when test="criterion.betweenValue">
                  and ${criterion.condition} #{criterion.value} and #{criterion.secondValue}
                </when>
                <when test="criterion.listValue">
                  and ${criterion.condition}
                  <foreach close=")" collection="criterion.value" item="listItem" open="(" separator=",">
                    #{listItem}
                  </foreach>
                </when>
              </choose>
            </foreach>
          </trim>
        </if>
      </foreach>
    </where>
  </sql>
  <sql id="Base_Column_List">
    "FEEDBACK_ID", "FEEDBACK_NAME", "FEEDBACK_CONENT", "FEEDBACK_TIME"
  </sql>
  <select id="selectByExample" parameterType="com.share.golden.entity.SysFeedbackExample" resultMap="BaseResultMap">
    select
    <if test="distinct">
      distinct
    </if>
    <include refid="Base_Column_List" />
    from SYS_FEEDBACK
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
    <if test="orderByClause != null">
      order by ${orderByClause}
    </if>
  </select>
  <select id="selectByPrimaryKey" parameterType="java.lang.String" resultMap="BaseResultMap">
    select 
    <include refid="Base_Column_List" />
    from SYS_FEEDBACK
    where "FEEDBACK_ID" = #{feedbackId,jdbcType=VARCHAR}
  </select>
  <delete id="deleteByPrimaryKey" parameterType="java.lang.String">
    delete from SYS_FEEDBACK
    where "FEEDBACK_ID" = #{feedbackId,jdbcType=VARCHAR}
  </delete>
  <delete id="deleteByExample" parameterType="com.share.golden.entity.SysFeedbackExample">
    delete from SYS_FEEDBACK
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
  </delete>
  <insert id="insert" parameterType="com.share.golden.entity.SysFeedback">
    insert into SYS_FEEDBACK ("FEEDBACK_ID", "FEEDBACK_NAME", "FEEDBACK_CONENT", 
      "FEEDBACK_TIME")
    values (#{feedbackId,jdbcType=VARCHAR}, #{feedbackName,jdbcType=VARCHAR}, #{feedbackConent,jdbcType=VARCHAR}, 
      #{feedbackTime,jdbcType=TIMESTAMP})
  </insert>
  <insert id="insertSelective" parameterType="com.share.golden.entity.SysFeedback">
    insert into SYS_FEEDBACK
    <trim prefix="(" suffix=")" suffixOverrides=",">
      <if test="feedbackId != null">
        "FEEDBACK_ID",
      </if>
      <if test="feedbackName != null">
        "FEEDBACK_NAME",
      </if>
      <if test="feedbackConent != null">
        "FEEDBACK_CONENT",
      </if>
      <if test="feedbackTime != null">
        "FEEDBACK_TIME",
      </if>
    </trim>
    <trim prefix="values (" suffix=")" suffixOverrides=",">
      <if test="feedbackId != null">
        #{feedbackId,jdbcType=VARCHAR},
      </if>
      <if test="feedbackName != null">
        #{feedbackName,jdbcType=VARCHAR},
      </if>
      <if test="feedbackConent != null">
        #{feedbackConent,jdbcType=VARCHAR},
      </if>
      <if test="feedbackTime != null">
        #{feedbackTime,jdbcType=TIMESTAMP},
      </if>
    </trim>
  </insert>
  <select id="countByExample" parameterType="com.share.golden.entity.SysFeedbackExample" resultType="java.lang.Long">
    select count(*) from SYS_FEEDBACK
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
  </select>
  <update id="updateByExampleSelective" parameterType="map">
    update SYS_FEEDBACK
    <set>
      <if test="record.feedbackId != null">
        "FEEDBACK_ID" = #{record.feedbackId,jdbcType=VARCHAR},
      </if>
      <if test="record.feedbackName != null">
        "FEEDBACK_NAME" = #{record.feedbackName,jdbcType=VARCHAR},
      </if>
      <if test="record.feedbackConent != null">
        "FEEDBACK_CONENT" = #{record.feedbackConent,jdbcType=VARCHAR},
      </if>
      <if test="record.feedbackTime != null">
        "FEEDBACK_TIME" = #{record.feedbackTime,jdbcType=TIMESTAMP},
      </if>
    </set>
    <if test="_parameter != null">
      <include refid="Update_By_Example_Where_Clause" />
    </if>
  </update>
  <update id="updateByExample" parameterType="map">
    update SYS_FEEDBACK
    set "FEEDBACK_ID" = #{record.feedbackId,jdbcType=VARCHAR},
      "FEEDBACK_NAME" = #{record.feedbackName,jdbcType=VARCHAR},
      "FEEDBACK_CONENT" = #{record.feedbackConent,jdbcType=VARCHAR},
      "FEEDBACK_TIME" = #{record.feedbackTime,jdbcType=TIMESTAMP}
    <if test="_parameter != null">
      <include refid="Update_By_Example_Where_Clause" />
    </if>
  </update>
  <update id="updateByPrimaryKeySelective" parameterType="com.share.golden.entity.SysFeedback">
    update SYS_FEEDBACK
    <set>
      <if test="feedbackName != null">
        "FEEDBACK_NAME" = #{feedbackName,jdbcType=VARCHAR},
      </if>
      <if test="feedbackConent != null">
        "FEEDBACK_CONENT" = #{feedbackConent,jdbcType=VARCHAR},
      </if>
      <if test="feedbackTime != null">
        "FEEDBACK_TIME" = #{feedbackTime,jdbcType=TIMESTAMP},
      </if>
    </set>
    where "FEEDBACK_ID" = #{feedbackId,jdbcType=VARCHAR}
  </update>
  <update id="updateByPrimaryKey" parameterType="com.share.golden.entity.SysFeedback">
    update SYS_FEEDBACK
    set "FEEDBACK_NAME" = #{feedbackName,jdbcType=VARCHAR},
      "FEEDBACK_CONENT" = #{feedbackConent,jdbcType=VARCHAR},
      "FEEDBACK_TIME" = #{feedbackTime,jdbcType=TIMESTAMP}
    where "FEEDBACK_ID" = #{feedbackId,jdbcType=VARCHAR}
  </update>
</mapper>